projects/plugins/jetpack/extensions/blocks/voice-to-content/edit.tsx

```diff
@@ -7,87 +7,214 @@ import {
 	playerPauseIcon,
 	useMediaRecording,
 } from '@automattic/jetpack-ai-client';
-import { useBlockProps } from '@wordpress/block-editor';
-import { Placeholder, Button } from '@wordpress/components';
+import { ThemeProvider } from '@automattic/jetpack-components';
+import { Button, Modal, Icon } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
 import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { external } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import oscilloscope from './assets/oscilloscope.svg';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function Oscilloscope( { audioURL } ) {
+	return <img src={ oscilloscope } alt="" />;
+}
+
+function ContextualRow( { state, error = null, audioURL = null } ) {
```

Comment:
Nitpicking: ContextualRow seems like a name that gives little context (pun intended), but honestly, I can't suggest anything better either. So I asked ChatGPT and it suggested AudioRecorderControlPanel. Just for you to consider.

Reply:
The reason for the name was that this can have either audio recording information, some notice or even an error. Naming this one ended up being weird.
I'll revisit this in a follow-up.